source/scripts/pages/EventPage/Balance.js

```diff
@@ -245,6 +230,16 @@ const BalancePage = React.createClass({
 						</Poster>
 					</FlexContainer>
 				}
+				{(positiveSum === 0 &&
+					negativeSum === 0 &&
+					othersDebts.length === 0 &&
+					returnedDebts.length !== 0) &&
```

Comment:
Works good!

There are a little opportunity for refactoring. Currently there are three possible states of the balance tab:

1. No debts at all
2. All debts are returned
3. There are some unreturned debts

For each state we render appropriate content like Poster or debts list. Currently rendering logic is very fragile because of duplicate conditions. We could extract rendering logic to a separate component method and clarify it by removing duplicate conditions.